daemon/graphdriver/overlay2/overlay.go

```diff
@@ -191,9 +191,12 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 		// Try to enable project quota support over xfs.
 		if d.quotaCtl, err = quota.NewControl(home); err == nil {
 			projectQuotaSupported = true
+		} else if opts.quota.Size > 0 {
+			logrus.Error("projectQuota Not supported")
+			return nil, err
```

Comment:
Do you suggest to have something like graphdriver.ErrPrjQuotaNotSupported ? or graphdriver.FeatureNotSupported ? 

Please suggest a name otherwise. Thanks :-)